keps/sig-cluster-lifecycle/kubeadm/20190925-component-configs.md

```diff
@@ -209,7 +211,7 @@ However, as this is only an initial version of the proposal and further changes
 
```

Comment:
re: graduation criteria ^
potentially add a note about graduating the new commands / flags in the Beta stage?

Reply:
I'll revise the KEP for the beta stage. In fact, I refer to this stage as "alpha 1" and I'll probably do an "alpha 2" one too (once the instance vs shared component configs KEP is in implementable state).